### ISSUE TITLE
spice-gtk-0.34 fails to install from nixpkgs-18.09pre131986.893b69a54cb

### ISSUE THREAD

**Issue by lierdakil** [NixOS/nixpkgs#37110]
## Issue description

`spice-gtk-0.34` fails to install complaining about missing `spice-client-glib-usb-acl-helper`

(more specifically, current `nixpkgs.spice-gtk`)

### Steps to reproduce

```
$ nix-env -i spice-gtk-0.34
warning: there are multiple derivations named 'spice-gtk-0.34'; using the first one
installing 'spice-gtk-0.34'
these paths will be fetched (1.78 MiB download, 9.41 MiB unpacked):
  /nix/store/1gv5mifakl7nfz3zymh89n3y9agc6ifn-spice-gtk-0.34
  /nix/store/b40d0299y8y8kxfvf4zn2b541f5yn7n6-usbredir-0.7.1
  /nix/store/h78l3k4172nmp0qx2pcisxpn6nc4rqkf-usbutils-009
  /nix/store/jid52565zlg8jhgqarln207dcj5g9njn-hwdata-0.309
  /nix/store/k6l7vajclwlds4cf8pc1d9gk34jk6jff-celt-0.5.1.3
copying path '/nix/store/k6l7vajclwlds4cf8pc1d9gk34jk6jff-celt-0.5.1.3' from 'https://cache.nixos.org'...
copying path '/nix/store/jid52565zlg8jhgqarln207dcj5g9njn-hwdata-0.309' from 'https://cache.nixos.org'...
copying path '/nix/store/b40d0299y8y8kxfvf4zn2b541f5yn7n6-usbredir-0.7.1' from 'https://cache.nixos.org'...
copying path '/nix/store/h78l3k4172nmp0qx2pcisxpn6nc4rqkf-usbutils-009' from 'https://cache.nixos.org'...
copying path '/nix/store/1gv5mifakl7nfz3zymh89n3y9agc6ifn-spice-gtk-0.34' from 'https://cache.nixos.org'...
building '/nix/store/37y4l41lyf9k9n8dszharcvfx2z3d56m-user-environment.drv'...
error: getting status of '/nix/store/1gv5mifakl7nfz3zymh89n3y9agc6ifn-spice-gtk-0.34/bin/spice-client-glib-usb-acl-helper': No such file or directory
builder for '/nix/store/37y4l41lyf9k9n8dszharcvfx2z3d56m-user-environment.drv' failed with exit code 1
error: build of '/nix/store/37y4l41lyf9k9n8dszharcvfx2z3d56m-user-environment.drv' failed
```

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.14.7-gentoo, Gentoo, noversion`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.0`
 - channels(root): `""`
 - channels(livid): `"nixpkgs-18.09pre131986.893b69a54cb"`
 - nixpkgs: `/home/livid/.nix-defexpr/channels/nixpkgs`

**Comment by xeji** [comment]
Thanks for your report. I could reproduce this and will look into it.

**Comment by xeji** [comment]
Fixed in #37184.
